evm-contracts/src/v0.4/interfaces/AggregatorV3Interface.sol

```diff
@@ -3,25 +3,25 @@ pragma solidity >=0.4.24;
 interface AggregatorV3Interface {
   function decimals() external view returns (uint8);
   function description() external view returns (string memory);
-  function getRoundData(uint256 _roundId)
+  function getRoundData(uint80 _roundId)
     external
     view
     returns (
-      uint256 roundId,
+      uint80 roundId,
```

Comment:
Cool, can this be any arbitrary number?

Reply:
Why uint80 over uint64 for example?

Reply:
We currently use a uint32 in the flux aggregator, but wanted to give ourselves some space to grow so went with a uint64. On top of that there's another uint16 for to denote phase packed in, adding up to uint80. We wanted to make it clear to consumers that they didn't need to use the whole 256 bits since we don't intend to, and they can possibly save some storage costs.